src/components/appSwitch/AppSwitch.js

```diff
@@ -238,8 +240,21 @@ const AppSwitch = (props: LoadingProps) => {
   }, [])
 
   useEffect(() => {
-    AppState.addEventListener('change', handleAppFocus)
+    if (isMobileNative && DeepLinking.pathname) {
+      return props.navigation.navigate(DeepLinking.pathname.slice(1))
+    }
+  }, [])
 
+  useEffect(() => {
+    if (isMobileNative && DeepLinking.pathname) {
+      DeepLinking.subscribe(() => {
+        return props.navigation.navigate(DeepLinking.pathname.slice(1))
+      })
+    }
+  }, [DeepLinking.pathname])
+
+  useEffect(() => {
+    AppState.addEventListener('change', handleAppFocus)
```

Comment:
the custom hook for handling AppState has been added by @sirpy 
pls look it in the master branch and re-use